### PR TITLE
Added egg component

### PR DESCRIPTION
**Pull request by theSaiRohit**

### CODE REVIEW

File: styles/modules/home/_styles.scss [hunk 1]
```diff
@@ -107,12 +107,12 @@ main {
...
-      height: 100%;
-      width: 100%;
-    }
+    // .egg-wrapper {
+    //   position: absolute;
+    //   top: 0;
+    //   left: 0;
+    //   height: 100%;
+    //   width: 100%;
+    // }
```
Comment (theGobindSingh) [inline styles/modules/home/_styles.scss:116]: Obsolete comments. Please remove.

**Review by theGobindSingh — CHANGES_REQUESTED** [review]
Please do changes according to the comments.

File: styles/modules/home/_styles.scss (hunk 1 above)
Comment (theSaiRohit) [reply styles/modules/home/_styles.scss]: Removed 👍

**Review by theGobindSingh — APPROVED** [review]
great work

**Pull request merged**